even-more-fish-plugin/src/main/java/com/oheers/fish/config/messages/Message.java

```diff
@@ -206,17 +202,13 @@ public String getString(String normal, String... ids) {
      *
      * @return The string list from config that matches the value of id.
      */
-    public List<String> getStringList(List<String> normal, String... ids) {
-        for (String id : ids) {
-            if (!Messages.getInstance().getConfig().contains(id)) continue;
-            List<String> returning = Messages.getInstance().getConfig().getStringList(id);
-            if (!returning.isEmpty()) return returning;
-            else {
-                EvenMoreFish.logger.log(Level.SEVERE, "No valid value in messages.yml for: " + id + " using default value instead.");
-                return normal;
-            }
+    public List<String> getStringList(List<String> normal, String id) {
+        List<String> list = Messages.getInstance().getConfig().getStringList(id);
+        if (list.isEmpty()) {
+            EvenMoreFish.logger.log(Level.SEVERE, "No valid value in messages.yml for: " + id + " using default value instead.");
```

Comment:
You can use a convinience function here. EvenMoreFish.getInstance().getLogger().severe("No valid value in messages.yml for: " + id + " using default value instead."). Instead of specifying the level.